content/1_General/Data_Types.mdx

```diff
@@ -162,9 +173,5 @@ both `cout << ('f' - 'a');` in C++ and `System.out.print('f' - 'a');` in Java
 will print `5`. In Java, characters are 16 bits, while in C/C++, characters are
 8 bits.
 
-**Strings** are stored as an array of characters. On the back end, there is an
-`ArrayList` (you can think of it as a more flexible array which can change its
-size if more space is needed), that automatically updates when two strings are
-added. There are a few methods such as `substring()` and `charAt()` in Java that
-are helpful to know. A loop can be used to access each character of an unknown
-string.
+**Strings** are effectively arrays of characters. You can easily access the character at a certain
+index and take substrings of the string (`charAt()` and `substring()` in Java).
```

Comment:
You can't really treat java strings like arraylists ... you can't set the `i`-th character of a string

Reply:
probably should make a note of this but I'm not a Java user so idrc